tests/csm/rest/test_capacity_usage.py

```diff
@@ -99,7 +100,7 @@ def setup_class(cls):
         cls.failed_pod = []
         cls.deployment_backup = None
         cls.fail_cnt = 0
-        cls.deploy_list = cls.master.get_deployment_name(cls.num_nodes)
+        cls.deploy_list = cls.master.get_deployment_name(POD_NAME_PREFIX)
```

Comment:
@kachhwahadivya please check once if this looks ok to you. cc: @priyankaborawake

Reply:
looks Ok